setup.cfg

```
@@ -54,15 +49,14 @@ install_requires =
     numpy==1.22.2; platform_machine=='loongarch64' and python_version<'3.11'
```

```suggestion
    numpy==1.22.2; platform_machine=='loongarch64' and python_version>='3.8' and python_version<'3.11'
```

numpy 1.22.2 does not support Python 3.7.